Dockerfile.README.md

```diff
@@ -39,4 +39,4 @@ If you are storing passwords for users, you should also run:
 openssl genpkey -out ./dockerdata/passkey.pem -outform PEM -algorithm RSA -pkeyopt rsa_keygen_bits:2048
 ```
 
-You can now run your shiny new image using `docker run -p 9999:9999 -v $PWD/dockerdata:/app/data`.
+You can now run your shiny new image using `docker run --publish 8090 --volume $PWD/dockerdata:/app/data`.
```

Comment:
As per https://github.com/matrix-org/matrix-appservice-irc/issues/1272 we've already hit issues when we changed the port to 9999. I would sooner change our default to 9999 than encounter this mess again :(

Reply:
I don't think we can ever change `DEFAULT_PORT` as that would change it for everyone who's not set it up in their config or CLI options.
https://github.com/matrix-org/matrix-appservice-irc/blob/73545600b848e816694ec2ebe0c33b65f7adc323/src/bridge/IrcBridge.ts#L47

That's why I am proposing to change the Docs.

Reply:
You're also changing the scripts / default config and lots of non doc things, in particular changing the docker start script will break existing setups (again), so we need to tread carefully here.

Reply:
Ok, I reverted most of that.
In docs/bridge_setup.md the port changed midway through the guide from 9999 to 9995, so I changed that to 9999.